examples/config.nims

```diff
@@ -1,22 +0,0 @@
---os:standalone
```

Comment:
Please keep the config (the flags part of it). It's allows using nim+clang more naturally unlike the makefile.

Reply:
there's a compile error when it's present that I haven't investigated, will readd later